tika/server.go

```diff
@@ -123,14 +123,14 @@ func (s *Server) Stop() error {
 	return nil
 }
 
-func md5Hash(path string) (string, error) {
+func getHash(path string, alg crypto.Hash) (string, error) {
```

Comment:
In general, avoid the "get" prefix in Go. https://golang.org/doc/effective_go.html#Getters

So, this would be `hash()`. But, if we only do sha512, this can be `sha512Hash()`.

Reply:
ok, I’ll fix it. if you don’t have to support such old version, I think it can be sha512Hash().